po/pl.po

```diff
@@ -756,9 +755,9 @@ msgstr "Oznaczam wszystkie kanały jako przeczytane..."
 
 #: src/feedlistformaction.cpp:434 src/feedlistformaction.cpp:962
 #: src/itemlistformaction.cpp:640 src/itemlistformaction.cpp:853
-#, fuzzy, c-format
+#, c-format
 msgid "Error: couldn't parse filter expression `%s': %s"
-msgstr "błąd składniowy komendy filtra `%s': %s"
+msgstr "Błąd:  nie udało się przetworzyć filtra `%s': %s"
```

Comment:
There's two spaces here, that'd be noticeable in the terminal with a monospace font. Can you please fix this?